test/util/generator_helpers.rb

```diff
@@ -0,0 +1,150 @@
+require "yaml"
+
+require 'beaker-hostgenerator'
+require 'beaker-hostgenerator/data/vmpooler'
+require 'beaker-hostgenerator/data'
+
+module GeneratorTestHelpers
+  include BeakerHostGenerator::Data
+  include BeakerHostGenerator::Data::Vmpooler
+
+  def run_cli_with_options options = []
```

Comment:
We should put the parens around the method parameters here and `open_file` below, for readability and consistency